Readme.md

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Nice in text, but not in `.md` format. You could start with ` ``` ` and end with ` ``` `, then it would be formatted as code.